src/Surfnet/StepupSelfService/SelfServiceBundle/Service/SmsSecondFactor/SmsVerificationState.php

```diff
@@ -86,6 +97,12 @@ public function requestNewOtp($phoneNumber)
      */
     public function verify($userOtp)
     {
+        if ($this->verificationAttemptsMade >= self::MAXIMUM_VERIFICATION_ATTEMPTS) {
+            return OtpVerification::tooManyAttempts();
```

Comment:
Maybe I'm missing something, but I can't see this working as the count is reset to `0` every instantiation and it doesn't seem to be stored anywhere? (the test checking this reuses the same instance, which is not the case in consecutive requests)